app/models/project.rb

```diff
@@ -1,3 +1,9 @@
 class Project < ApplicationRecord
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def self.count_projects
```

Comment:
For this user story, we were looking for the number of contestants on a given project (to display on the project show page). 
Rather than the number of total projects in the database, consider instead an instance method that counts contestants like so: `self.contestants.count` where self is the instance of the project

Reply:
Keep in mind for next time as well: if you're creating a model method, please create a model test for it as well!